services/common/src/components/comments/MinistryCommentPanel.tsx

```diff
@@ -1,46 +1,43 @@
-import React from "react";
-import PropTypes from "prop-types";
-import { UserOutlined, LoadingOutlined } from "@ant-design/icons";
-import { Avatar, Divider, Row, Col, Spin, List } from "antd";
+import React, { FC } from "react";
+import { LoadingOutlined, UserOutlined } from "@ant-design/icons";
+import { Avatar, Col, Divider, List, Row, Spin } from "antd";
 
 import CommentEditor from "@mds/common/components/comments/CommentEditor";
 import MinistryComment from "@mds/common/components/comments/MinistryComment";
-import * as Style from "@mds/common/constants/styles";
-import AuthorizationWrapper from "@mds/common/wrappers/AuthorizationWrapper";
+import * as Style from "@mds/core-web/src/constants/styles";
+import AuthorizationWrapper from "@mds/core-web/src/components/common/wrappers/AuthorizationWrapper";
```

Comment:
Strange imports!
I can see importing the style from core-web, but it makes me wonder if this component belongs in common since it's only used on core-web. But the AuthorizationWrapper I don't think we want to maintain a separate file for core-web

Reply:
Yeah, it's causing all kinds of problems.  Didn't look at them too carefully at first, but I'm working on cleaning that up now.